addons/base_vat/models/res_partner.py

```diff
@@ -77,14 +77,18 @@
 
 _region_specific_vat_codes = {
     'xi',
+    't',
 }
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     def _split_vat(self, vat):
-        vat_country, vat_number = vat[:2].lower(), vat[2:].replace(' ', '')
+        if vat[1].isalpha():
+            vat_country, vat_number = vat[:2].lower(), vat[2:].replace(' ', '')
```

Comment:
add commentary: we might have a single letter to determine the country (in japan, T blablabla)